test/functionalTest/cases/0970_create_entity/empty_attribute_value.test

```diff
@@ -65,19 +65,21 @@ Location: /v2/entities/ID
 Date: REGEX(.*)
 
 
+
```

Comment:
This test has been "recovered" (from .test.DISABLED to .test).

NTC (just a remark)